task4/punkapi/Краткое описание.txt

```diff
@@ -0,0 +1,5 @@
+https://api.punkapi.com/v2/beers/1 - возвращает информацию о пиве с id 1
+https://api.punkapi.com/v2/beers?abv_gt2&ibu_gt=2 - возвращает информацию о пиве, подходящее под параметры
+abv_gt - пиво должно иметь ABV больше, чем 2
```

Comment:
Not-a-mistake, but please take care to give  more detailed info about your code when drilling a commercial project, A reader, or a reviewer, might need to know, that, for instance, IBU is the measure of bitterness (_in International Bitterness Units_) and not the "International Biathlon Union" or sort of)